{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"summary"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":2,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/authfoundation\/oktaapierror\/summary"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation\/OktaAPIError\/summary","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"A short description of what caused this error. Sometimes this contains dynamically-generated information about your specific error."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"summary"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"title":"summary","roleHeading":"Instance Property","role":"symbol","symbolKind":"property","externalID":"s:14AuthFoundation12OktaAPIErrorV7summarySSvp","modules":[{"name":"AuthFoundation"}]},"hierarchy":{"paths":[["doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation","doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation\/OktaAPIError"]]},"references":{"doc://com.okta.mobile.authfoundation/documentation/AuthFoundation/OktaAPIError/summary":{"role":"symbol","title":"summary","fragments":[{"kind":"keyword","text":"let"},{"kind":"text","text":" "},{"kind":"identifier","text":"summary"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"abstract":[{"type":"text","text":"A short description of what caused this error. Sometimes this contains dynamically-generated information about your specific error."}],"identifier":"doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation\/OktaAPIError\/summary","kind":"symbol","type":"topic","url":"\/documentation\/authfoundation\/oktaapierror\/summary"},"doc://com.okta.mobile.authfoundation/documentation/AuthFoundation/OktaAPIError":{"role":"symbol","title":"OktaAPIError","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"OktaAPIError"}],"abstract":[{"type":"text","text":"Describes errors that may be reported from an Okta API endpoint."}],"identifier":"doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation\/OktaAPIError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"OktaAPIError"}],"url":"\/documentation\/authfoundation\/oktaapierror"},"doc://com.okta.mobile.authfoundation/documentation/AuthFoundation":{"role":"collection","title":"AuthFoundation","abstract":[{"type":"text","text":"Library that provides foundational features used by Oktaâ€™s authentication libraries, as well as APIs used to work with tokens and user credentials."}],"identifier":"doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation","kind":"symbol","type":"topic","url":"\/documentation\/authfoundation"}}}